### PR TITLE
Document feature flag 'created-timestamp-zero-ingestion'

### PR DESCRIPTION
**Pull request by ArthurSens**
Documenting the feature flag added in #12733

### CODE REVIEW

File: docs/feature_flags.md [hunk 1]
```diff
@@ -194,3 +194,15 @@ won't work when you push OTLP metrics.
...
+
+`--enable-feature=created-timestamp-zero-ingestion`
+
+Enables ingestion of Created Timestamps as injection of samples with 0 as value. 
+
+Created timestamps are only supported by the traditional Prometheus Protobuf protocol. When enabling this feature,
+therefore, the default scrape protocol will be switched to Prometheus Protobuf.
+
+Besides enabling this feature in Prometheus, created timestamps need to be exposed by the application being scraped.
+So far, only Prometheus Go Client has support for it.
```
Comment (bwplotka) [inline docs/feature_flags.md:208]:
```suggestion
```
Perhaps I would remove this part as it's easy to change soon and we will for sure forget to update this piece of documentation (:

File: docs/feature_flags.md [hunk 2]
```diff
@@ -194,3 +194,15 @@ won't work when you push OTLP metrics.
 
 Enables PromQL functions that are considered experimental and whose name or
 semantics could change.
+
+## Created Timestamps Zero Injection
+
+`--enable-feature=created-timestamp-zero-ingestion`
+
+Enables ingestion of Created Timestamps as injection of samples with 0 as value.
```
Comment (bwplotka) [inline docs/feature_flags.md:202]:
```suggestion
Enables ingestion of created timestamps as a injection of samples with 0 value for created timestamp (when appropriate). See PromCon talk <link> for details on the created timestamp.
```

File: docs/feature_flags.md [hunk 3]
```diff
@@ -194,3 +194,15 @@ won't work when you push OTLP metrics.
...
 semantics could change.
+
+## Created Timestamps Zero Injection
+
+`--enable-feature=created-timestamp-zero-ingestion`
+
+Enables ingestion of Created Timestamps as injection of samples with 0 as value. 
+
+Created timestamps are only supported by the traditional Prometheus Protobuf protocol. When enabling this feature,
+therefore, the default scrape protocol will be switched to Prometheus Protobuf.
```
Comment (bwplotka) [inline docs/feature_flags.md:205]:
```suggestion
```suggestion
Currently Prometheus supports created timestamp only on the traditional Prometheus Protobuf protocol (WIP for other protocols). As a result, when enabling this feature, the Prometheus protobuf scrape protocol will be prioritized (See `scrape_config.scrape_protocols` settings for more details).
```

**Review by bwplotka — COMMENTED** [review]
Nice! Some suggestions 💪🏽

File: docs/feature_flags.md (hunk 1 above)
Comment (ArthurSens) [reply docs/feature_flags.md:208]: yeah fair point

File: docs/feature_flags.md [hunk 4]
```diff
@@ -194,3 +194,13 @@ won't work when you push OTLP metrics.
 
 Enables PromQL functions that are considered experimental and whose name or
 semantics could change.
+
+## Created Timestamps Zero Injection
+
+`--enable-feature=created-timestamp-zero-ingestion`
+
+Enables ingestion of created timestamps as a injection of samples with 0 value for created timestamp (when appropriate). See PromCon talk <https://youtu.be/nWf0BfQ5EEA?feature=shared> for details on the created timestamp.
```
Comment (bwplotka) [inline docs/feature_flags.md:202]:
```suggestion
Enables ingestion of created timestamp. Created timestamps are injected as 0 valued samples when appropriate. See [PromCon talk](https://youtu.be/nWf0BfQ5EEA) for details.
```

Comment (bwplotka) [reply docs/feature_flags.md:202]: Don't treat my suggestions literally @ArthurSens (: `<Link>` was just semantically what would be nice, not literal structure of link in markdown (:

**Review by bwplotka — APPROVED** [review]
Nice, last nit! Thanks.

File: docs/feature_flags.md (hunk 4 above)
Comment (ArthurSens) [reply docs/feature_flags.md:202]: My bad, for some reason I thought you were saying that this is how links are shown in the website 😅

**Review by bwplotka — APPROVED** [review]

**Pull request merged**